### PR TITLE
landing: add FAQ section

### PR DESCRIPTION
**Pull request by Diogomartf**
image

### CODE REVIEW

File: packages/landing/app/page.tsx [hunk 1]
```diff
@@ -71,18 +73,99 @@ export default function Home() {
...
+                  investment strategies, it should be carefully considered in
+                  the context of one's financial situation and objectives.`}
+                </p>
+                <p className="text-lg font-medium text-em-med">
+                  {`Ultimately, DCA is a strategy that can help individuals build
+                  wealth steadily over time while reducing the risks associated
+                  with market timing and volatility. However, like all
+                  investment strategies, it should be carefully considered in
+                  the context of one's financial situation and objectives.`}
+                </p>
```
Comment (berteotti) [inline packages/landing/app/page.tsx:160]: Too much text here, right?

File: packages/landing/components/QandA/QandA.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import { Icon } from "@/ui";
+import { cx } from "class-variance-authority";
+import { PropsWithChildren, useState } from "react";
+
+interface QandAProps extends PropsWithChildren {
+  q: string;
+}
+
+export const QandA = ({ q, children }: QandAProps) => {
```
Comment (berteotti) [inline packages/landing/components/QandA/QandA.tsx:11]: Maybe call it `QuestionAccoridon` or `Q&AAccoridon` or something similar. I feel this name is too vague.

Comment (Diogomartf) [reply packages/landing/components/QandA/QandA.tsx:11]: Q&A usually represents question and answer, that I feel represents what this is better than the suggestions you gave.

File: packages/landing/app/page.tsx (hunk 1 above)
Comment (Diogomartf) [reply packages/landing/app/page.tsx:160]: I think it's good for now. 

We'll do text review on the polishing phase when we have the whole website.

File: packages/landing/components/QandA/QandA.tsx (hunk 2 above)
Comment (berteotti) [reply packages/landing/components/QandA/QandA.tsx:11]: I just think it's too generic. We can give it a little more context with Accordion.

Comment (berteotti) [reply packages/landing/components/QandA/QandA.tsx:11]: And maybe `QAndAAccordion`. QandA it's a weird pattern

Comment (Diogomartf) [reply packages/landing/components/QandA/QandA.tsx:11]: I'll take the `QAndAAccordion`

**Comment by berteotti** [comment]
Can give approval after conflicts are solved

**Review by berteotti — APPROVED** [review]

**Pull request merged**